### PR TITLE
fix(rn): Provide a way to clean duplicated foreign keys

### PR DESCRIPTION
**Pull request by danila-m**
Provide a way to clean duplicated foreign keys caused by
RUNTIME-154/HHH-13779

Relates to [RUNTIME-154](https://bonitasoft.atlassian.net/browse/RUNTIME-154)

### CODE REVIEW

File: modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -179,12 +179,30 @@ Take a look at the xref:define-a-search-index.adoc[search keys documentation] to
...
 Previously, the strategy to generate table ID's of BDM objects was left to Hibernate to decide. It could be a database SEQUENCE, an auto-incremented column, ... +
 Now, the specific implementation is explicitly set on each ID column during the BDM class generation. It is however different
 from one Database vendor to another (a database SEQUENCE for Oracle and PostgreSQL, an auto-incremented column for MySQL and MS SQL Server). +
 Note that no change is required from you, and upgrading to Bonita 2021.1 does not affect your already generated BDM until you actually redeploy it (next time you update it).
 
+===== Foreign key generation
+In Bonita 7.11, the automatic name generation for foreign keys changed. It does not affect new BDM deployments, or already deployed BDMs.
+However, because of a bug (RUNTIME-154, itself caused by the bug HHH-13779 in the underlying hibernate library), redeploying a BDM causes those foreign key to be generated a second time.
+On Oracle, it is forbidden to generate such duplicated foreign keys, so the BDM redeployment operation fails. This is fixed in the next Bonita 2021.1 release.
```
Comment (pabloalonso) [inline modules/ROOT/pages/release-notes.adoc]: This is in the 2021.1 RN so we should explain that this is fixed in the 2021.1-0408 "patch" version
```suggestion
On Oracle, it is forbidden to generate such duplicated foreign keys, so the BDM redeployment operation fails. This is fixed in the latest Bonita 2021.1 release (2021.1-0408)
```

File: modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -179,12 +179,30 @@ Take a look at the xref:define-a-search-index.adoc[search keys documentation] to
...
+On other supported Db vendors the resulting schema in the database will have some duplicated foreign keys: foreign keys with a different name, but otherwise identical.
+Note: it will never generate more than two identical foreign keys, even after several BDM redeployment.
+This has no effect on the behavior of either the Bonita platform, or the BDM. That said, you may wish to clean your BDM database of these duplicated foreign keys. To do so:
+
+1. Stop your bonita server
+
+1. Open the db in an edition tool (or execute in command line the relevant commands)
```
Comment (pabloalonso) [inline modules/ROOT/pages/release-notes.adoc]: We should explain how to find the duplicated FKs and how to remove them.
What about a community question?

**Review by pabloalonso — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/release-notes.adoc (hunk 2 above)
Comment (pabloalonso) [reply modules/ROOT/pages/release-notes.adoc]: move step by step with SQL commands into migrate-from-an-earlier-version-of-bonita-bpm

**Comment by alachambre** [comment]
No activity for 2 months on this pull request, could you guys finihs it or drop it? @danila-m @pabloalonso

**Comment by educhastenier** [comment]
This PR is duplicate of https://github.com/bonitasoft/bonita-doc/pull/1512.
=> Closing

**Pull request closed without merging**